x-pack/plugins/aiops/public/components/change_point_detection/fields_config.tsx

```diff
@@ -248,6 +249,7 @@ const FieldPanel: FC<FieldPanelProps> = ({
             disabled: removeDisabled,
           },
         ],
+        'data=test-subj': 'aiopsChangePointDetectionContextMenuPanel',
```

Comment:
The `=` should be `-` I guess?

Reply:
Agreed with @darnautov to fix in a separate PR as this needs correcting on `main` too, and this `data-test-subj` is not currently being used.